src/datastore/DistanceBetweenExpansionStadium.csv

```diff
@@ -1,4 +1,4 @@
 # id, stadium name, id, stadium name, distance
-80,Las Vegas Stadium,52,Oakland-Alameda County Coliseum,325
-80,Las Vegas Stadium,55,Dodger Stadium,300
-80,Las Vegas Stadium,61,Chase Field,250
\ No newline at end of file
+80,Las Vegas Stadium,69,Oakland-Alameda County Coliseum,325
+80,Las Vegas Stadium,63,Dodger Stadium,300
+80,Las Vegas Stadium,50,Chase Field,250
```

Comment:
Why were the stadium IDs changed?

Reply:
updated to match MLBInfo file

Reply:
Why were they changed in the first place?